oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -92,6 +92,10 @@
   private static final long serialVersionUID = 7807543542681217978L;
   private static final String GRANT_TYPE = "urn:ietf:params:oauth:grant-type:jwt-bearer";
   private static final String PARSE_ERROR_PREFIX = "Error parsing token refresh response. ";
+  private static final int TWELVE_HOURS_IN_SECONDS = 43200;
+  private static final int DEFAULT_LIFETIME_IN_SECONDS = 3600;
+  private static final String LIFETIME_EXCEEDED_ERROR =
```

Comment:
From the other style PR, inline the error message.

Reply:
done